articles/connections/passwordless/sms-gateway.md

```diff
@@ -117,39 +141,4 @@ If the SMS Gateway returns an HTTP code other than 200, the `/passwordless/start
 
 If the SMS Gateway returns HTTP 401, the `error_description` will be **Authentication failed while calling the SMS gateway: 401**.
 
-Note that the `error_description` field is not part of the endpoint contract and is subject to change. 
-
-## Configure an authenticated SMS Gateway
-
-The previous settings assume your SMS Gateway accepts non-authenticated requests. The `sms_gateway` provider also allows you to configure token-based authentication (using `gateway_authentication`):
-
-```
-{
-    "options": {
-      "strategy": "sms",
-      "provider": "sms_gateway",
-      "gateway_url": "{URL_OF_YOUR_GATEWAY}",
-      "gateway_authentication": {
-            "method": "bearer",
-            "subject": "urn:Auth0",
-            "audience": "urn:MySmsGateway",
-            "secret": "MySecretToSignTheToken"
-      },
-      "from": "+1 234 567",
-      "template": "Your verification code is: @@password@@",
-      "brute_force_protection": true,
-      "forward_req_info": "true",
-      "disable_signup": false,
-      "name": "sms",
-      "syntax": "md_with_macros",
-      "totp": {
-        "time_step": 300,
-        "length": 6
-      }
-    }
-}
-```
-
-With this configuration, when the payload is sent to the SMS Gateway, a JWT will be added to the `Authorization` header which contains a token with the `subject` and `audience` configured in the connection and signed with the `secret`.
-
-Additionally, if your secret is base64-url-encoded, you can set `options.gateway_authentication.secret_base64_encoded` to `true`.
+**The `error_description` field is not part of the endpoint contract and is subject to change.**
```

Comment:
What is an endpoint contract? What are we actually telling them? Not to use error_description? If so, why are we telling them what it is? Is there something more useful we should suggest instead? I'd be frustrated if I saw this at the end of a doc because I would be reading to be told what to do for error catching, and that isn't really what we're doing here right now.